### PR TITLE
FAQ: Antivirus

### PR DESCRIPTION
**Pull request by psiinon**
<details><summary>Screenshot</summary>
<p>

Screenshot 2024-05-17 at 10-56-29 ZAP – Why does my Antivirus Tool Flag ZAP

</p>
</details>

### CODE REVIEW

File: site/content/faq/why-does-my-antivirus-tool-flag-zap.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+The [Active Scan Rule add-on](/docs/desktop/addons/active-scan-rules/) contains the rules which attack websites, 
+so it is not surprising that make AV tools flag it.
+
+Any issues raised about ZAP or its add-ons being flagged by AV tools will be closed with a link to this FAQ.
+
+If you have hard evidence that there is an actual virus in ZAP or its add-ons then please report that with the evidence to 
+the ZAP [Bug Bounty Program](https://bugcrowd.com/owaspzap).
+
+If you work for an AV tool vendor and would like to discuss how you can make sure your tool does not incorrectly flag ZAP then please
+[get in touch](mailto:support@zaproxy.com).
```
Comment (kingthorin) [inline site/content/faq/why-does-my-antivirus-tool-flag-zap.md:32]: I'd say move this up to line 10 😀

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

File: site/content/faq/why-does-my-antivirus-tool-flag-zap.md (hunk 1 above)
Comment (psiinon) [reply site/content/faq/why-does-my-antivirus-tool-flag-zap.md:32]: I know what you mean, but the "For example.." paragraph doesnt make as much sense then?

Comment (kingthorin) [reply site/content/faq/why-does-my-antivirus-tool-flag-zap.md:32]: Move it to the very top and put a heading in before "we know", or even turn "we know" into like a 2, 3, 4 level heading?

Comment (psiinon) [reply site/content/faq/why-does-my-antivirus-tool-flag-zap.md:32]: The main target audience for this FAQ will be ZAP users. How about I leave this line where it is but make it bold, or a heading?

Comment (kingthorin) [reply site/content/faq/why-does-my-antivirus-tool-flag-zap.md:32]: Sure!

Comment (psiinon) [reply site/content/faq/why-does-my-antivirus-tool-flag-zap.md]: Made it **bold** :)

**Review by kingthorin — APPROVED** [review]

**Pull request merged**